build_tools/cmake/easy.sh

```diff
@@ -0,0 +1,291 @@
+#!/bin/bash
+
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# A CMake wrapper for concise command lines and stateless operation.
```

Comment:
Could add a note here about the intended audience: developers frequently changing configuration or using cross compilation, **not** downstream projects.

Reply:
This is going to be clarified by not having this script in-tree.